app/src/main/res/values/strings.xml

```diff
@@ -21,21 +20,14 @@
   <string name="settings_status_on">Stop</string>
   <string name="settings_status_off_summary">Service stopped</string>
   <string name="settings_status_on_summary">Service running</string>
-  <string name="settings_secure_title">Encryption</string>
-  <string name="settings_secure_off">Enable</string>
-  <string name="settings_secure_on">Disable</string>
-  <string name="settings_secure_off_summary">Encryption disabled</string>
-  <string name="settings_secure_on_summary">Encryption enabled</string>
   <string name="settings_provider_title">Location provider</string>
   <string name="settings_provider_summary">Source of location data</string>
   <string name="settings_provider_gps">GPS provider</string>
   <string name="settings_provider_network">Network provider</string>
   <string name="settings_provider_mixed">Mixed provider</string>
   <string name="settings_foreground_title">Foreground service</string>
   <string name="settings_foreground_summary">Increase service priority</string>
-  <string name="error_invalid_address">Invalid server address</string>
-  <string name="error_invalid_port">Invalid server port</string>
-  <string name="error_invalid_frequency">Invalid frequency value</string>
+  <string name="settings_invalid_url_summary">Invalid server URL</string>
```

Comment:
Please use `error_` prefix for both new strings and put them next to each other in the file.